clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -469,7 +470,7 @@ private void processDisconnection(List<ClientResponse> responses, String nodeId,
         nodeApiVersions.remove(nodeId);
         nodesNeedingApiVersionsFetch.remove(nodeId);
         for (InFlightRequest request : this.inFlightRequests.clearAll(nodeId)) {
-            log.trace("Cancelled request {} due to node {} being disconnected", request, nodeId);
+            log.trace("Cancelled request {} due to node {} being disconnected", request.requestBuilder, nodeId);
```

Comment:
Minor but I'm not sure if we'd prefer the builder's toString or the underlying request struct's toString as was formerly the case.